docs/index.md

```diff
@@ -15,1621 +15,118 @@ parent="smn_workw_machine"
 > don't recommend you use it in production yet.
 
 Machine lets you create Docker hosts on your computer, on cloud providers, and
-inside your own data center. It creates servers, installs Docker on them, then
-configures the Docker client to talk to them.
+inside your own data center. It automatically creates hosts, installs Docker on
+them, then configures the `docker` client to talk to them.  A "machine" is the
+combination of a Docker Host and a configured client.
 
-Once your Docker host has been created, it then has a number of commands for
-managing them:
+Once you create one or more Docker hosts, Docker Machine supplies a number of commands for
+managing them. Using these commands you can 
 
- - Starting, stopping, restarting
- - Upgrading Docker
- - Configuring the Docker client to talk to your host
+ - start, inspect, stop, and restart a host
+ - upgrade the Docker client and daemon
+ - configure a Docker client to talk to your host
 
-## Getting help
-
-Docker Machine is still in its infancy and under active development. If you need
-help, would like to contribute, or simply want to talk about to the project with
-like-minded individuals, we have a number of open channels for communication.
-
-- To report bugs or file feature requests: please use the [issue tracker on
-  Github](https://github.com/docker/machine/issues).
-- To talk about the project with people in real time: please join the
-  `#docker-machine` channel on IRC.
-- To contribute code or documentation changes: please [submit a pull request on
-  Github](https://github.com/docker/machine/pulls).
-
-For more information and resources, please visit
-[https://docs.docker.com/project/get-help/](https://docs.docker.com/project/get-help/).
 
-## Installation
+## A simple Docker Machine workflow
```

Comment:
Using something like a machine is what is known as an "ill structured" problem. The first step in training that kind of material is to outline the concepts and basic procedure.  The purpose of this section and the larger page is to explain the Machine component concepts and workflow -- it is exposition not procedure. 

This type of teaching was largely missing from the original documentation which launched into using the terms provider and driver interchangeably without explaining what they were and how they related.  Similarly,  in the original, base operating system was used without a full explanation --- until the last section of the page.

The local VM page is actually the next step in teaching --- it goes further to actually ask the user to have the components install and walk through a problem. Effectively I am build a set of ever more sophisticated problems to teach users.